seacrowd/sea_datasets/vsolscsum/vsolscsum.py

```diff
@@ -0,0 +1,244 @@
+# limitations under the License.
+
+"""
+The Vietnamese dataset for social context summarization \
+    The dataset contains 141 open-domain articles along with \
+    3,760 sentences, 2,448 extracted standard sentences and \
+    comments as standard summaries and 6,926 comments in 12 \
+    events. This dataset was manually annotated by human. \
+    Note that the extracted standard summaries also include comments.\
+    The label of a sentence or comment was generated based on the \
+    voting among social annotators. For example, given a sentence, \
+    each annotator makes a binary decision in order to indicate \
+    that whether this sentence is a summary candidate (YES) or not \
+    (NO). If three annotators agree yes, this sentences is labeled by 3. \
+    Therefore, the label of each sentence or comment ranges from 1 to 5\
+    (1: very poor, 2: poor, 3: fair, 4: good; 5: perfect). The standard \
+    summary sentences are those which receive at least three agreements \
+    from annotators. The inter-agreement calculated by Cohen's Kappa \
+    after validation among annotators is 0.685.
+"""
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+import xml.etree.ElementTree as ET
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@inproceedings{nguyen-etal-2016-vsolscsum,
+    title = "{VS}o{LSCS}um: Building a {V}ietnamese Sentence-Comment Dataset for Social Context Summarization",
+    author = "Nguyen, Minh-Tien  and
+      Lai, Dac Viet  and
+      Do, Phong-Khac  and
+      Tran, Duc-Vu  and
+      Nguyen, Minh-Le",
+    editor = "Hasida, Koiti  and
+      Wong, Kam-Fai  and
+      Calzorari, Nicoletta  and
+      Choi, Key-Sun",
+    booktitle = "Proceedings of the 12th Workshop on {A}sian Language Resources ({ALR}12)",
+    month = dec,
+    year = "2016",
+    address = "Osaka, Japan",
+    publisher = "The COLING 2016 Organizing Committee",
+    url = "https://aclanthology.org/W16-5405",
+    pages = "38--48",
+    abstract = "This paper presents VSoLSCSum, a Vietnamese linked sentence-comment dataset, which was manually created to treat the lack of standard corpora for social context summarization in Vietnamese. The dataset was collected through the keywords of 141 Web documents in 12 special events, which were mentioned on Vietnamese Web pages. Social users were asked to involve in creating standard summaries and the label of each sentence or comment. The inter-agreement calculated by Cohen{'}s Kappa among raters after validating is 0.685. To illustrate the potential use of our dataset, a learning to rank method was trained by using a set of local and social features. Experimental results indicate that the summary model trained on our dataset outperforms state-of-the-art baselines in both ROUGE-1 and ROUGE-2 in social context summarization.",
```

Comment:
The abstract is also mentioned in the `_DESCRIPTION`, the description below class (line 110-127) and the beginning (line 16-33). Let's remove this line

Reply:
Solved on https://github.com/SEACrowd/seacrowd-datahub/pull/609/commits/18815e7622ee0d853fd0d903a3366a26d9015b86.